app/controllers/change_requests_controller.rb

```diff
@@ -43,6 +43,13 @@ def approve
     else
       change_request = ChangeRequest.find params[:change_request_id]
       if change_request.pending?
+
+        FieldChange.delete_all(["change_request_id = ?", change_request.id])
+
+        change_request.field_changes = field_changes_approve change_request.id
+
+        change_request.save!
```

Comment:
For safety, perhaps surround this set of changes with a transaction so that in case if an error occurs, we don't end up deleting the old field changes without creating the new ones?